.travis/prepare.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ex
+
+# Stop any services that bind to ports we need.
+case $CHECK in
+    couchbase|mcache)
+        sudo service memcached stop
+        while sudo lsof -Pi :11211 -sTCP:LISTEN -t; do sleep 1; done
+        ;;
+    pgbouncer|postgres)
```

Comment:
I'm afraid this will have to be maintained, shouldn't we just change the ports we're using in the tests?

Reply:
I tried that but some of the underlying Dockerfiles seem to expose ports themselves.